smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/middleware/MiddlewareExecutionGenerator.kt

```diff
@@ -86,12 +86,27 @@ class MiddlewareExecutionGenerator(
         renderMiddlewares(ctx, op, operationStackName)
 
         if (ctx.settings.useInterceptors) {
+            val rpcService = symbolProvider.toSymbol(serviceShape).getName().removeSuffix("Client")
+            val rpcMethod = op.getId().getName()
             writer.write(
                 """
+                var metricsAttributes = ${"$"}N()
+                metricsAttributes.set(key: ${"$"}N.service, value: ${"$"}S)
+                metricsAttributes.set(key: ${"$"}N.method, value: ${"$"}S)
```

Comment:
Generate RPC service / method attributes that are standardized (not `*Client` and not lowercased operation name).

RPC service / method attributes are commonly used in metrics emitted in Orchestrator.